src/Resources/app/administration/src/module/sw-cms/elements/twitter-timeline/config/sw-cms-el-config-twitter-timeline.html.twig

```diff
@@ -1,11 +1,43 @@
 {% block sw_cms_element_twitter_timeline_config %}
-    <div class="sw-cms-el-config-twitter-timeline">
+<div class="sw-cms-el-config-twitter-timeline">
+    {% block sw_cms_element_twitter_timeline_config_twitter_handle %}
         <sw-field class="twitter-handle-field"
                   v-model="element.config.twitterHandle.value"
                   type="text"
-                  label="Twitter Timeline"
-                  placeholder="Enter Twitter Handle."
-                  @blur="onBlur">
+                  :label="$tc('sw-cms.elements.twitterHandle.label')"
+                  :placeholder="$tc('sw-cms.elements.twitterHandle.placeHolder')">
         </sw-field>
-    </div>
+    {% endblock %}
+
+    {% block sw_cms_element_twitter_timeline_config_timeline_type %}
+    <sw-select-field v-model="element.config.timelineType.value"
+                     :label="$tc('sw-cms.elements.twitterTimeline.type.label')"
+                     :placeholder="$tc('sw-cms.elements.twitterTimeline.type.placeHolder')">
+        <option value="profile">{{ $tc('sw-cms.elements.twitterTimeline.type.profile') }}</option>
+        <option value="profileLikes">{{ $tc('sw-cms.elements.twitterTimeline.type.profileLikes') }}</option>
+        <option value="collections">{{ $tc('sw-cms.elements.twitterTimeline.type.collections') }}</option>
+        <option value="lists">{{ $tc('sw-cms.elements.twitterTimeline.type.lists') }}</option>
+    </sw-select-field>
+    {% endblock %}
+
+    {% block sw_cms_element_twitter_timeline_config_timeline_list %}
+    <sw-field class="timeline-list-field"
+              v-if="element.config.timelineType && element.config.timelineType.value === 'lists'"
```

Comment:
The condition logic should be a function and inside `src/Resources/app/administration/src/module/sw-cms/elements/twitter-timeline/config/index.js`

Reply:
I had it before like that, but it is so an easy condition I decided to have it direct in the template